jans-cedarling/cedarling/src/authz/entities/create.rs

```diff
@@ -10,26 +10,28 @@ use std::{
     str::FromStr,
 };
 
-use crate::common::{
-    cedar_schema::{
-        cedar_json::{CedarSchemaEntityShape, CedarSchemaRecord, CedarType, GetCedarTypeError},
-        CedarSchemaJson,
-    },
-    policy_store::ClaimMappings,
-};
+use super::trait_as_expression::AsExpression;
+use crate::jwt::{TokenClaim, TokenClaimTypeError, TokenData};
 use crate::{
-    authz::token_data::{GetTokenClaimValue, Payload, TokenPayload},
-    common::cedar_schema::cedar_json::SchemaDefinedType,
+    common::{
```

Comment:
It would be easier to read if the imports were more flattened.

Reply:
changed in [2051f1c](https://github.com/JanssenProject/jans/pull/10436/commits/2051f1c570f4b538c6e476768f36a58a31ab5b0c)